installers/vmware/kickstart-esxi.go

```diff
@@ -308,18 +307,17 @@ kickstartpishellcmd=$(custom_data "['kickstart']['postinstall_shell_cmd']")
 
 # Kickstart postinstall supplemental config URL
 if [ "$kickstartpiurl" != "null" ]; then
-        echo "Using supplemental kickstart postinstall URL: $kickstartpiurl"
-        if wget -q -s $kickstartpiurl; then
-                wget -q $kickstartpiurl -O /tmp/ks-postinstall-sup.sh
-                echo "========Begin execution of supplemental postinstall kickstart"
-                chmod +x /tmp/ks-postinstall-sup.sh && /tmp/ks-postinstall-sup.sh
-                echo "========End execution of supplemental postinstall kickstart"
-        else
-                echo "ERROR: Custom kickstart postinstall URL is NOT accessible!"
-                exit 1
-        fi
+	echo "Using supplemental kickstart postinstall URL: $kickstartpiurl"
+	if wget -q "$kickstartpiurl" -O /tmp/ks-postinstall-sup.sh
```

Comment:
this line is going to need a "; then" suffix.

Reply:
gah! copy/paste fail in github editor :(

Reply:
good catch! updated